### PR TITLE
[Refactor:TAGrading] Reorder Grading Pages

### PR DESCRIPTION
**Pull request by cjreed121**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
Right now when you click grade on an assignment, you are first directed to the status page which then you can go to the student index.

### What is the new behavior?
Clicking the grade button takes you right to the student index page. There is a grading stats button which can take you to that page. There is a new pop up on the student index for the responsibilities of graders. This message normally shows on the status page but that is being skipped so it will show here. It only shows the first time for each gradeable once you agree. Not agreeing will take you to course home page.

This uses localStorage to know if you have viewed the grading page before.

### CODE REVIEW

**Review by williamjallen — CHANGES_REQUESTED** [review]
The code looks good, except for one small section of inline CSS which was copied from the original page.  I do have a few concerns about the UI though.
1. When viewing a teams gradeable, the "Export Teams members" button is on a separate line from the rest of the buttons. 
Untitled
2. In my opinion, the "Grading Stats" and "Responsibilities as a Grader" buttons should be secondary buttons, not primary buttons.
3. Perhaps we can discuss the exact arrangement of buttons later/make a future PR but to me, the page seems somewhat cluttered with buttons, particularly on the Instructor UI, and I wonder if it could be cleaned up somewhat.

File: site/app/templates/grading/electronic/GradeableMessage.twig [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+<div hidden data-semester="{{ semester }}" data-course="{{ course }}" data-gradeable="{{ gradeable_id }}" data-userlevel="{{ core.getUser().getGroup() }}" id="gradeable-message-data"></div>
+
+<!-- If grader is limited access or full access grader -->
+{% if core.getUser().getGroup() == 2 or core.getUser().getGroup() == 3 %}
+    <div class="content gradeable_message">
+        <p>
+            You will be reviewing materials submitted by students and these materials are considered confidential.<br>
+            The author of this material is entitled to copyright protection. As such,
+        <ul>
+            <li style="margin-left:30px;margin-bottom:10px;">You may not use or share any of these materials or ideas without explicit permission from the author.</li>
```
[the remark this answers is not in the document]
Comment (cjreed121) [reply site/app/templates/grading/electronic/GradeableMessage.twig:15]: Wouldn't that just then apply those styles to the whole <ul> tag and not the actual <li> tags? This is what I had copied from the previous page so this is what exists already. I could throw the CSS in details.css instead if you think that's a better option.

**Review by williamjallen — APPROVED** [review]

**Comment by bmcutler** [comment]
@DefinitiveAbove please review this...
Related to open PR #6689, aren't we getting rid of the 'view=all'?

**Review by DefinitiveAbove — CHANGES_REQUESTED** [review]
Looks good to me for the most part; there's still a bit of inline CSS for one of the cancel buttons on the popup.

I'm not sure if it should be changed in this PR, but the "Export Teams Members" button is still on a separate line from the rest of the buttons as noted earlier.

> Related to open PR #6689, aren't we getting rid of the 'view=all'?

Yes; I'm not sure if it'd be better to remove it here or later in PR #6689.

**Comment by bmcutler** [comment]
The new page sequence is much improved!

**Pull request merged**